<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountCreation</class>
 <widget class="QDialog" name="AccountCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Creation</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create Your Account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Username :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_username">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>University :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_uv">
        <item>
         <property name="text">
          <string>Northern University Bangladesh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>North South University Bangladesh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brac University Bangladesh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Department :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_dept">
        <item>
         <property name="text">
          <string>CSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EEE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Others</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Date of Birth :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_dob"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Captcha :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_captcha">
        <property name="text">
         <string>0+0 =</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_captcha"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Agree with our T&amp;C :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_agree">
        <property name="text">
         <string>I'm Agree</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Login">
      <property name="text">
       <string>Create Account</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
